tailwind.config.ts

```diff
@@ -0,0 +1,19 @@
+import type { Config } from 'tailwindcss';
+
+export default {
+  content: ['./src/**/*.{js,jsx,ts,tsx}'],
+  theme: {
+    screens: {
+      sm: '576px',
+      md: '780px',
+      lg: '780px',
```

Comment:
Large could be laptop-sized? Esp as we might need a big difference between web and mobile

Maybe even an xl

Reply:
Np, that was a copy/paste job tbf. I just updated that now though :)